types/web-bluetooth/index.d.ts

```diff
@@ -5,121 +5,121 @@
+export interface BluetoothDeviceEventHandlers {
+        onadvertisementreceived: (this: this, ev: Event) => any;
+        ongattserverdisconnected: (this: this, ev: Event) => any;
+    }
 
-interface BluetoothDevice extends EventTarget, BluetoothDeviceEventHandlers, CharacteristicEventHandlers, ServiceEventHandlers {
-	readonly id: string;
-	readonly name?: string;
-	readonly gatt?: BluetoothRemoteGATTServer;
-	readonly uuids?: string[];
-	watchAdvertisements(): Promise<void>;
-	unwatchAdvertisements(): void;
-	readonly watchingAdvertisements: boolean;
-	addEventListener(type: "gattserverdisconnected", listener: (this: this, ev: Event) => any, useCapture?: boolean): void;
-	addEventListener(type: "advertisementreceived", listener: (this: this, ev: Event) => any, useCapture?: boolean): void;
-	addEventListener(type: string, listener: EventListenerOrEventListenerObject, useCapture?: boolean): void;
-}
+export interface BluetoothDevice extends EventTarget, BluetoothDeviceEventHandlers, CharacteristicEventHandlers, ServiceEventHandlers {
+        readonly id: string;
+        readonly name?: string;
+        readonly gatt?: BluetoothRemoteGATTServer;
+        readonly uuids?: string[];
+        watchAdvertisements(): Promise<void>;
+        unwatchAdvertisements(): void;
+        readonly watchingAdvertisements: boolean;
+        addEventListener(type: "gattserverdisconnected", listener: (this: this, ev: Event) => any, useCapture?: boolean): void;
+        addEventListener(type: "advertisementreceived", listener: (this: this, ev: Event) => any, useCapture?: boolean): void;
+        addEventListener(type: string, listener: EventListenerOrEventListenerObject, useCapture?: boolean): void;
+    }
 
-interface Bluetooth extends EventTarget, BluetoothDeviceEventHandlers, CharacteristicEventHandlers, ServiceEventHandlers {
-	getAvailability(): Promise<boolean>;
-	onavailabilitychanged: (this: this, ev: Event) => any;
-	readonly referringDevice?: BluetoothDevice;
-	requestDevice(options?: RequestDeviceOptions): Promise<BluetoothDevice>;
-	addEventListener(type: "availabilitychanged", listener: (this: this, ev: Event) => any, useCapture?: boolean): void;
-	addEventListener(type: string, listener: EventListenerOrEventListenerObject, useCapture?: boolean): void;
-}
+export interface Bluetooth extends EventTarget, BluetoothDeviceEventHandlers, CharacteristicEventHandlers, ServiceEventHandlers {
+        getAvailability(): Promise<boolean>;
+        onavailabilitychanged: (this: this, ev: Event) => any;
+        readonly referringDevice?: BluetoothDevice;
+        requestDevice(options?: RequestDeviceOptions): Promise<BluetoothDevice>;
+        addEventListener(type: "availabilitychanged", listener: (this: this, ev: Event) => any, useCapture?: boolean): void;
+        addEventListener(type: string, listener: EventListenerOrEventListenerObject, useCapture?: boolean): void;
+    }
 
-interface Navigator {
-	bluetooth: Bluetooth;
-}
+export interface Navigator {
```

Comment:
Does this merge cleanly with the main navigator type

Reply:
Well, tbh, I am in no way able to use navigator.bluetooth without some tricks. I could also not run the tests with npm test, until I did the fixes as in this PR.
How is this supposed to work at all?
This does work, with and without export, and only if I import Navigator from web-bluetooth:
```
import { Navigator } from 'web-bluetooth';
const navigator = window.navigator as unknown as Navigator;
navigator.bluetooth.requestDevice(options)...
```
Or, this without the import
```
const navigator: any = window.navigator;
navigator.bluetooth.requestDevice(options)..
```

Not sure what the intended usage is here..